src/olympia/abuse/cinder.py

```diff
@@ -197,6 +203,13 @@ def create_job_decision(self, *, action, reasoning, policy_uuids, job_id):
         url = f'{settings.CINDER_SERVER_URL}jobs/{job_id}/decision'
         return self._send_create_decision(url, {}, action, reasoning, policy_uuids)
 
+    def create_override_decision(self, *, action, reasoning, policy_uuids, decision_id):
+        url = f'{settings.CINDER_SERVER_URL}decisions/{decision_id}/override/'
+        # TODO: send action too once
+        # https://lindie.app/share/6a21d831b39351d7c6fe898f6d22619af62dde98/PLAT-1834
+        # implements the same parameters for overrides
```

Comment:
Can we track this in a ticket on our side as well?

Reply:
https://github.com/mozilla/addons/issues/15371